[13.0][ADD] account_payment_term_security: New module (#577)
OCA/account-payment: account_payment_term_security/README.rst

   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--payment-lightgray.png?logo=github
    :target: https://github.com/OCA/account-payment/tree/13.0/account_payment_term_security
    :alt: OCA/account-payment
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/account-payment-13-0/account-payment-13-0-account_payment_term_security
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/96/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> In partners and invoices, payment terms can only be managed when the user has permission "Enable to manage payment terms".

Reviewer: ```suggestion
In partners and invoices, payment terms can only be changed when the user has permission "Change payment terms in partners and invoices".
```

**Table of contents**

.. contents::
   :local:

Configuration
=============

To use this module, you need to :

> Go to Settings > Users & Companies > Users (edit your user) > Set "Enable to manage payment terms" option.

Reviewer: ```suggestion
Go to Settings > Users & Companies > Users (edit your user) > Set "Change payment terms in partners and invoices" option.
```
